react/src/components/CreateMapModal/CreateMapModal.module.css

```diff
@@ -11,10 +11,54 @@
 
 .custom-error-message {
   margin-top: 0.25rem;
-  font-size: 0.8em;
+  font-size: 1.2em;
   color: #dc3545;
 }
 
 .form-check-label {
   font-style: italic;
 }
+
+.input-custom-size {
+  width: 35.75rem;
+}
+
+.flexContainer {
+  display: flex;
+  vertical-align: top;
+  gap: 8px;
+  margin-bottom: 1.5em;
+}
+
+.flexContainer-alt {
+  display: flex;
+  vertical-align: top;
+  gap: 8px;
+  margin-bottom: 1.5em;
+  margin-top: -1.5rem;
+}
+
+.check-wrapper {
+  display: flex;
+  margin-left: 1rem;
+  margin-top: -0.9rem;
+}
+
+.hazmapper-custom {
+  display: inline-flex;
+  align-items: center;
+  padding: 0 0.75rem;
+  font-size: 1rem;
+  background-color: #e9ecef;
+  border: 1px solid #ced4da;
+  border-radius: 0.25rem;
+  margin-top: 2.27rem;
+  margin-left: -9px;
+  min-width: 100px;
+  height: 2.9em;
+}
```

Comment:
I updated an outdated screenshot to reflect the current state of the Hazmapper UI.

Reply:
I recognized and added a link to [Bootstrap's Input Group feature](https://getbootstrap.com/docs/4.0/components/input-group/). If it easier to adapt `<Formik…>` wrappers to support that, I'll try not to unleash the Wesley who wants Bootstrap to burn in hell, and rationally consider it.

Reply:
I got the idea from Bootstrap's input group feature, but I know the Form/FormField comp from core-components has an add-on parameter I was using before. I can go back to using the FormField component and use that add-on if you're ok with that instead of FormikInput (at least for that field, I can still use FormikInput for the Name field).

Reply:
Alternatively I could add an "add-on" prop to the FormikInput wrapper too.

Reply:
I do like the look of the s-affixed-input-wrapper--suffix ($) in that example. If we can do that with .hazmapper instead of the dollar sign, that would look good I think.

Reply:
**Yes**, the dollar sign can be replaced with ".hazmapper". **Note:** this modal form's specific instance of usage must set `--width` to a value that tucks the suffix into the field.

<img width="1194" alt="Screenshot 2024-02-21 at 6 47 40 PM" src="https://github.com/TACC-Cloud/hazmapper/assets/62723358/c97659ec-e8ab-47a1-86f6-e9300c9412a1">

I'd prefer code be added to our Formik components to support this, but I haven't thought about the workload. I also haven't considered "[I can go back to using the FormField component](https://github.com/TACC-Cloud/hazmapper/pull/208#discussion_r1498212706)", that may be more efficient.

Would you please try to "go back to using the FormField component" if successful, and offer both options to designers in #cep-design? You may use my screenshot for one. _Even if they want to use the Core-Styles one, I wouldn't judge if you used a separate ticket to convert from Bootstrap to Core-Styles for this field._

Reply:
I plan to make a new task for this change, as well, and address it early March.

Reply:
Created a task for the March sprint [WG-264](https://tacc-main.atlassian.net/browse/WG-264)